pkg/remote/remote.go

```diff
@@ -61,6 +61,8 @@ FROM {{ .UserRunnerImage }} as runner
 USER 0
 {{ end -}}
 ENV PATH="${PATH}:/okteto/bin"
+RUN if [ -d /okteto ]; then echo "/okteto folder is reserved for internal use"; exit 1; fi
```

Comment:
Creating `/okteto` in the image is a valid use case. For example, if you need to install node modules for your test container, you have to do it in your test image at the `/okteto/src` location

Reply:
Also, can we remove the `{{ if .UseRootUser -}}` logic now?